### PR TITLE
Added kernelversion fact for Ubuntu.

### PR DESCRIPTION
**Pull request by joelmheim**
Should now show proper kernel version number on Ubuntu. Uses the Uname resolver and parses out the version numbers from that string.

Added a few tests to make sure that the parsing is handling a few cases.

### CODE REVIEW

**Comment by joelmheim** [comment]
Fixes issue #104 .

**Comment by sebastian-miclea** [comment]
> Should now show proper kernel version number on Ubuntu. Uses the Uname resolver and parses out the version numbers from that string.
> 
> Added a few tests to make sure that the parsing is handling a few cases.

Thank you for your contribution!

File: lib/facts/ubuntu/kernelversion.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Facter
+  module Ubuntu
+    class Kernelversion
+      FACT_NAME = 'kernelversion'
+
+      def call_the_resolver
+        version_numbers = Resolvers::Uname.resolve(:kernelrelease).scan(/\d+/)
+        fact_value = version_numbers[0..2].join '.'
```
Comment (sebastian-miclea) [inline lib/facts/ubuntu/kernelversion.rb]: Could you please add brackets to the join method?

Comment (joelmheim) [reply lib/facts/ubuntu/kernelversion.rb]: Will do. Sorry for the late reply. Have been travelling.

**Comment by joelmheim** [comment]
Added parens to join. Sorry about the delay.

**Review by sebastian-miclea — APPROVED** [review]

**Pull request merged**